modules/core/index.js

```diff
@@ -0,0 +1,10 @@
+export { Connection } from './connection';
+export { Difference } from './difference';
+export { Entity } from './entity';
+export { Graph } from './graph';
+export { History } from './history';
+export { iD_Node as Node } from './node';
```

Comment:
Does anything actually break if you export it as `Node` in both places?   I feel like the modules should scope their exports somehow so that they don't really conflict with anything in practice.  

(e.g. I think Browserify would that `Node` into something like `11_Node`)